src/service/viewport-impl.js

```diff
@@ -850,12 +879,16 @@ export class ViewportBindingNatural_ {
   }
 
   /** @override */
-  updatePaddingTop(paddingTop, opt_updateScrollPos, opt_lastPaddingTop) {
+  updatePaddingTop(paddingTop) {
     this.win.document.documentElement.style.paddingTop = px(paddingTop);
-    if (opt_updateScrollPos) {
-      const oldScrollTop = this.getScrollTop();
-      this.setScrollTop(oldScrollTop + paddingTop - opt_lastPaddingTop);
-    }
+  }
+
+  /** @override */
+  hideViewerHeader(unusedLastPaddingTop) {
```

Comment:
@dima In natural binding case, should I update padding && adjust scroll position when `transient == true` ?